### ISSUE TITLE
Reproduce Llava-uhd Evaluations?

### ISSUE THREAD

**Issue by Andrew-Zhang** [ParadoxZW/LLaVA-UHD-Better#7]
Hello! Thanks so much for fixing the bugs in the Llava-uhd repo. I was wondering if anyone was able to reproduce the evaluations that Llava-uhd got in their paper. I also noticed that my pretraining converges to a loss ~3 whereas the [original Llava pretraining](https://wandb.ai/lht/huggingface/reports/LLaVA-v1-5-Training-Logs--Vmlldzo1NzE1Mzcx?accessToken=gh6ju8v40olw6d7p9ja498ntyxrddbzy23xovhde3hc8nk5dva2jy46u5hj0dg2u) converges to ~2. I this expected behavior?
 
Thanks,
Andrew

**Comment by ParadoxZW** [comment]
I've also struggled in reproducing the results that could match the paper. QAQ
